caboose.go

```diff
@@ -268,6 +272,10 @@ func (c *Caboose) Has(ctx context.Context, it cid.Cid) (bool, error) {
 }
 
 func (c *Caboose) Get(ctx context.Context, it cid.Cid) (blocks.Block, error) {
+	fetchCalledTotalMetric.WithLabelValues(resourceTypeBlock).Add(1)
+	if recordIfContextErr(resourceTypeBlock, ctx, "FetchBlockApi") {
+		return nil, ctx.Err()
+	}
```

Comment:
this is only a partial view of this, right? - because asks for `Has` and `GetSize` also trigger parallel block triggers to `Get`?

Reply:
@willscott Yeah, just wanna focus on fetch for now.

Reply:
then lets not record the block api at all - misreporting will lead to confusion?

Reply:
@willscott 

Fixed the instrumentation. We now record on the lower level block and car fetchBlockWith and fetchResourceWith instead of the public API for capturing accurate numbers.